src/openagi/agent.py

```diff
@@ -420,12 +439,78 @@ def single_agent_execution(self, query: str, description: str, task_lists: TaskL
         logging.debug(f"Execution Completed for Session ID - {self.memory.session_id}")
         return output
 
+
     def run(self, query: str, description: str,planned_tasks: Optional[List[Dict]] = None):
         logging.info("Running Admin Agent...")
         logging.info(f"SessionID - {self.memory.session_id}")
 
-        if planned_tasks is None:
-            planned_tasks = self.run_planner(query=query, descripton=description)
+        if self.memory.long_term and planned_tasks:
+            logging.warning("Long Term Memory is not applicable for user given plan.")
+
+        """
```

Comment:
if the docstring is not required, please discard it

Reply:
yep, discarded